site/.editorconfig

```diff
@@ -2,15 +2,11 @@ root = true
 
 [*]
 indent_style = space
-indent_size = 2
+indent_size = 4
```

Comment:
We indent HTML, CSS and JS files with two spaces (iirc).

Reply:
We use 2 for html, css, js and 4 elsewhere where not specified